### ISSUE TITLE
Getting rid of softmax activations

### ISSUE THREAD

**Issue by sebastian-lapuschkin** [albermax/innvestigate#85]
[graph.py](https://github.com/albermax/innvestigate/blob/master/innvestigate/utils/keras/graph.py) provides a method `model_wo_softmax()` which creates a new model based on a given model by routing its inputs and outputs to the input of the given model and the pre-softmax-outputs of the given model.
(Short version: It takes the network graph and uses appropriate in- and outputs)

This function makes use of the method `pre_softmax_tensors`, also in `graph.py`, but is missing the **s** when calling `pre_softmax_tensors`. 
See `graph.py:308`
